### PR TITLE
Registering Team[The Coding Hush]

### PR DESCRIPTION
**Pull request by richhaa**
# Application for Registering Team to CODE CAMP 1.0

This pull request template helps you complete an application to the **CODE CAMP 1.0 HACKATHON**. Use the checklist below to verify that you have followed the instructions correctly. 

**(Put x inside [] like [x])**

## Select the type of Pull Request

- [] Registration Pull request.

- [] Submission Pull request.

## Checklist ✅  

- [] I have read the instructions on the README file before submitting my application and gone through [CODE_OF_CONDUCT](). 
- [] I have Submitted my team's details by making a folder of my team as instructed in [How to send Pull Request page](https://codecamphackathon.netlify.app/makingRegistration/).
- [] I have used the Markdown file template to add my information for the Hackathon.
- [] I have given all the necessary details as mentioned in template index.md file and the details are correct and best to my knowledge.
- [] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [] I understand I should not tag or add a reviewer to this Pull Request.
- [] I understand the Details added to the template will be used as a means of communication at the time of result declaration.
- [] I have added the [event](https://calendar.google.com/calendar/r/eventedit/copy/MHUzM2pvaDQwM3R1cDdxMmdjMmQyYjZpaHAgN3NhMzI0N2FpMzkzczAzbWo0YWMxNWVzdG9AZw) to my calendar.

### CODE REVIEW

File: content/teams/DemoTeam/The_Coding_Hush.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
+
+---
+type: "team"                                                        
+title: The Coding Hush
+name: "The Coding Hush"
+github-link: "https://github.com/ "
```
Comment (Uyadav207) [inline content/teams/DemoTeam/The_Coding_Hush.md]: Give your github link ( Leaders github link only )

File: content/teams/DemoTeam/The_Coding_Hush.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+_**University-Name**_ : Mumbai University
+
+_**country**_ : India
+ 
+_**Technical-Skills**_ : Python, Java, Oracle, C, Html5, CSS3, flask, javascript, React Js, php
+
+_**Specialization**_ :Java, Oracle, Front end development
+
+_**GitHub-ID**_ :  richhaa
```
Comment (Uyadav207) [inline content/teams/DemoTeam/The_Coding_Hush.md:85]: give complete link for all github links.

**Review by Uyadav207 — CHANGES_REQUESTED** [review]

File: content/teams/DemoTeam/The_Coding_Hush.md (hunk 2 above)
Comment (Uyadav207) [reply content/teams/DemoTeam/The_Coding_Hush.md:85]: here for all individual member, give their github link

Comment (Uyadav207) [reply content/teams/DemoTeam/The_Coding_Hush.md:85]: Also fill the check boxes thoroughly

File: content/teams/DemoTeam/The_Coding_Hush.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+                           we do understand it and hence, this web application is our way of empathizing with
+                           them.
+
+
+
+
+_**Tech-Stack-Used**_ : Python, Java, Html5, CSS3, flask, React Js, php, javascript, sql lite3, MySql, 
+                         Oracle, sql server 
+
+_**GitHub-Link**_ :   Github project Repository link Goes here (make a blank repository).
```
Comment (Uyadav207) [inline content/teams/DemoTeam/The_Coding_Hush.md:61]: Mention your hackathon project blank repository here. ( Link required )

check listing not done.

**Review by Uyadav207 — CHANGES_REQUESTED** [review]

**Comment by richhaa** [comment]
Are we suppose to put the github link or github id for every individual?

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by Uyadav207** [comment]
@richhaa , Mention github link for every individual and check list the above application.

also mention github blank repository for your Project.

**Comment by Uyadav207** [comment]
@richhaa .. The pull request you are submitting is not appropriate.

please follow the naming conventions and file directory as mentioned.

[Here](https://codecamphackathon.netlify.app/makingRegistration/)

## problems facing

- file directory is wrong.

## solution
- make a new fork 
- make a copy of `DemoTeam` among the teams. [Pay attention at this part]
- Edit the copied Demo Team as per your naming conventions.
- And do not touch any template just put your information only with complete links and project blank repository as told earlier 

SUBMIT YOUR REGISTRATION PULL REQUEST AGAIN

For now I am closing this PR

**Comment by richhaa** [comment]
Could you please check and revert if everything is appropriate
 this time

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Pull request closed without merging**